docs/docs/introduction.md

```diff
@@ -4,17 +4,17 @@ sidebar_position: 0
 
 # Introduction
 
-This is a specification for the packaging, installation, update, and monitoring process of [Eigen Layer](https://www.eigenlayer.xyz/) Middleware services. Middleware is used to participate in the Eigen Layer restaking system, to know more about Eigen Layer see its [documentation page](https://docs.eigenlayer.xyz/overview/readme). 
+This is a specification for the packaging, installation, update, and monitoring process of [Eigen Layer](https://www.eigenlayer.xyz/) AVSs. Middleware is used to participate in the Eigen Layer restaking system, to know more about Eigen Layer see its [documentation page](https://docs.eigenlayer.xyz/overview/readme). 
 
 The current version of the specification is `v1.0.0`.
 
 Documentation is divided into the following sections:
 
-- [Setup wizard](/docs/wizard/intro): A Middleware wizard setup tool, aimed at easing Middleware service establishment.
+- [Setup wizard](/docs/wizard/intro): A Middleware wizard setup tool, aimed at easing AVS establishment.
```

Comment:
Middleware wizard -> node setup tool
You can do this change in a different PR as well